src/SFA.DAS.ManageApprenticeships/Application/Queries/Transfers/GetFinancialBreakdown/GetFinancialBreakdownHandler.cs

```diff
@@ -23,25 +24,39 @@ public GetFinancialBreakdownHandler(IForecastingApiClient<ForecastingApiConfigur
 
         public async Task<GetFinancialBreakdownResult> Handle(GetFinancialBreakdownQuery request, CancellationToken cancellationToken)
         {
-            var transferFinancialBreakdownTask = _forecastingApiClient.Get<GetTransferFinancialBreakdownResponse>
-                                                        (new GetTransferFinancialBreakdownRequest(request.AccountId));
-            
-            var pledgesTask = _levyTransferMatchingApiClient.Get<GetPledgesResponse>
-                                                        (new GetPledgesRequest(request.AccountId));
+            var breakdownTask = await _forecastingApiClient.Get<GetTransferFinancialBreakdownResponse>
+                                                        (new GetTransferFinancialBreakdownRequest(request.AccountId, DateTime.UtcNow));
 
-            await Task.WhenAll(transferFinancialBreakdownTask, pledgesTask);
+            var nextYearBreakdowntask = await _forecastingApiClient.Get<GetTransferFinancialBreakdownResponse>
+                                                        (new GetTransferFinancialBreakdownRequest(request.AccountId, DateTime.UtcNow.AddYears(1)));
 
+            var yearAfterNextYearBreakdowntask = await _forecastingApiClient.Get<GetTransferFinancialBreakdownResponse>
+                                                        (new GetTransferFinancialBreakdownRequest(request.AccountId, DateTime.UtcNow.AddYears(2)));
```

Comment:
The tasks are being awaited separately and could be awaited together to improve performance.

Reply:
I have added await Task.WhenAll